src/App.js

```diff
@@ -1,5 +1,83 @@
-import HomePage from './pages/HomePage';
-
-export default function App() {
-  return <HomePage />;
-}
+import {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import HomePage from './pages/HomePage';
+import NewTodoForm from './components/NewTodoForm';
+import {loadFromLocalStorage, writeToLocalStorage} from './util/localstorage';
+import {dailyTodos} from './db';
+
+const templatesFromLocal = loadFromLocalStorage('TaskTemplates');
+const todosFromLocal = loadFromLocalStorage(new Date().toLocaleDateString());
+
+export default function App() {
+  const [taskTemplates] = useState(() => {
+    return templatesFromLocal || dailyTodos;

```

Comment:
I thought that the logical OR `||` returns either `true` or `false`, so `taskTemplates is either `true` or `false`, but has never the value of `templatesFromLocal` nor `dailyTodos`?

Reply:
I see, and it works as expected this way? I was thinking about the [nullish coalescing operator](https://javascript.info/nullish-coalescing-operator) `??`.

Reply:
Now I understand :D It did not work. Thanks for the hint!